### DISCUSSION TITLE
How to compile and install from sourcecode ?

### DISCUSSION THREAD

**Discussion by pedrognsmartins** [langflow-ai/langflow#1138] [Q&A]
Can I ask you kindly to support me on how to compile and install from sourcecode on Windows OS?

Thanks for the support

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
